pkg/kube/inject/inject_test.go

```diff
@@ -66,7 +66,8 @@ func TestInjection(t *testing.T) {
 			want: "hello.yaml.cni.injected",
 			setFlags: []string{
 				"components.cni.enabled=true",
-				"values.istio_cni.chained=true",
+				"values.cni.enabled=true",
```

Comment:
@howardjohn I was seeing some behavior where the values were not properly propagating without this config (see previous commit [here](https://github.com/istio/istio/pull/45826/commits/229b9a3958d4fca0124eeed6cb1f5cd713c09732#diff-1be653d6b63bcfb9a82318073d22f283f20a273dcb32ad94122929795e21ad6fL18)) This may need some minor configuration changes to make this consistent throughout all of the openshift configurations, because the helm templates don't recognize the component setting.  Just wanted to point this out because it may be confusing.